src/main/java/io/kestra/plugin/notifications/telegram/TelegramTemplate.java

```diff
@@ -0,0 +1,62 @@
+package io.kestra.plugin.notifications.telegram;
+
+import com.google.common.base.Charsets;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.tasks.VoidOutput;
+import io.kestra.core.runners.RunContext;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+        title = "Send input as a telegram message"
+)
+public class TelegramTemplate extends TelegramSend {
```

Comment:
I think this class cannot be used directly so better to make it abstract and remove its `@Schema` annotation (but keep those of the properties)